modules/oadp-release-notes-1-2-3.adoc

```diff
@@ -34,5 +34,12 @@ For a complete list of all issues resolved in the release of OADP 1.2.3, see the
 [id="known-issues-1-2-3_{context}"]
 == Known issues
 
-There are no known issues in the release of OADP 1.2.3.
+The {oadp-short} 1.2.3 has the following known issue:
 
+. Data Protection Application (DPA) does not reconcile when the credentials secret is updated
```

Comment:
When you fixed the abbreviation, an extra space was added, and now this renders as a numbered bullet.

![image](https://github.com/openshift/openshift-docs/assets/82119979/d9820040-ce8d-4ad4-8363-ab4308b1e84f)

As long as no other rendering errors creep in, this LGTM and I can merge.